packages/hts_shrink/hts_shrink.2.1.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/hts_shrink"
+bug-reports: "https://github.com/UnixJunkie/hts_shrink/issues"
+dev-repo: "git+https://github.com/UnixJunkie/hts_shrink.git"
+license: "BSD-3-Clause"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune"
```

Comment:
Could the `dune-project` file be included in the source for the next release? I think dune 1.0 is fine here but it might change in the future

Reply:
Ok, the dune-project file was added to the repository upstream.